website/docs/r/vpc_ipam_pool.html.markdown

```diff
@@ -69,7 +69,7 @@ resource "aws_vpc_ipam_pool_cidr" "child_test" {
 
 This resource supports the following arguments:
 
-* `address_family` - (Optional) The IP protocol assigned to this pool. You must choose either IPv4 or IPv6 protocol for a pool.
+* `address_family` - (Required) The IP protocol assigned to this pool. You must choose either IPv4 or IPv6 protocol for a pool.
```

Comment:
This likely needs to be updated too.
https://github.com/hashicorp/terraform-provider-aws/blob/main/internal/service/ec2/ipam_pool.go#L49